stdlib/DelimitedFiles/test/runtests.jl

```diff
@@ -217,7 +217,7 @@ end
             "Yugoslavia (Cyrillic)", "Djordje Balasevic", "Југославија", "Ђорђе Балашевић",
             "Yugoslavia (Latin)", "Djordje Balasevic", "Jugoslavija", "Đorđe Balašević"]
 
-        i18n_arr = permutedims(reshape(i18n_data, 4, Int(floor(length(i18n_data)/4))), [2, 1])
+        i18n_arr = permutedims(reshape(i18n_data, 4, Int(fld(length(i18n_data),4))), [2, 1])
```

Comment:
If we keep this one, you can remove the call to `Int` since the result will be of type `Int` already.